src/Apps/Search/Routes/Artworks/SearchResultsArtworks.tsx

```diff
@@ -2,34 +2,34 @@ import { Box } from "@artsy/palette"
 import { SearchResultsArtworks_viewer } from "__generated__/SearchResultsArtworks_viewer.graphql"
 import { FilterState } from "Apps/Search/FilterState"
 import { SearchResultsFilterFragmentContainer as ArtworkGrid } from "Apps/Search/Routes/Artworks/Components/Filter/SearchResultsFilterContainer"
-import { Location } from "found"
+import { Location, Router } from "found"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Provider } from "unstated"
 
 export interface Props {
   viewer: SearchResultsArtworks_viewer
   location: Location
+  router: Router
 }
 
 export class SearchResultsArtworksRoute extends React.Component<Props> {
   render() {
-    const {
-      viewer,
-      location: { query },
-    } = this.props
+    const { viewer, location, router } = this.props
+    const { query } = location
     const { term } = query
 
     return (
       <Provider
         inject={[
           new FilterState({
+            ...query,
```

Comment:
This was key to get the filter UI to respect the URL params. We were already returning `location.query` in `prepareVariables` so they were making the correct request to MP w/ proper variables, but to get the UI to reflect that we need to inject it into the state.